### PR TITLE
* Add data-base-url attribute in body tag closes https://github.com/plone/Products.CMFPlone/2051

### PR DESCRIPTION
**Pull request by rodfersou**
* Fix @@plone_lock_operations fixes https://github.com/plone/Products.CMFPlone/issues/2061
* Review @@sharing fixes https://github.com/plone/Products.CMFPlone/2053

### CODE REVIEW

File: Products/CMFDefault/skins/ursine/main_template.pt [hunk 1]
```diff
@@ -3,15 +3,18 @@
   &quot;-//W3C//DTD XHTML 1.0 Transitional//EN&quot;
   &quot;http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd&quot;&gt;" />
 
-<html tal:define="globals context/@@ursine_globals"
+<html tal:define="globals context/@@ursine_globals;
+                  portal_state context/@@plone_portal_state;
+                  portal_url portal_state/portal_url"
```
Comment (tseaver) [inline Products/CMFDefault/skins/ursine/main_template.pt]: The `@@plone_portal_state` view is not defined in CMFDefault.  Instead, used `globals/portal_url`.

File: Products/CMFDefault/skins/werebear/main_template.pt [hunk 2]
```diff
@@ -3,8 +3,9 @@
   &quot;-//W3C//DTD XHTML 1.0 Transitional//EN&quot;
   &quot;http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd&quot;&gt;" />
 
-<html tal:define="globals context/@@ursine_globals"
->
+<html tal:define="globals context/@@ursine_globals;
+                  portal_state context/@@plone_portal_state;
```
Comment (tseaver) [inline Products/CMFDefault/skins/werebear/main_template.pt]: Ditto here.

File: Products/CMFDefault/skins/zpt_generic/main_template.pt [hunk 3]
```diff
@@ -4,6 +4,7 @@
   &quot;http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd&quot;&gt;" />
 
 <html tal:define="globals context/getMainGlobals;
+                  plone_view context/@@plone;
```
Comment (tseaver) [inline Products/CMFDefault/skins/zpt_generic/main_template.pt]: There is no `@@plone` view in CMFDefault.

**Review by tseaver — CHANGES_REQUESTED** [review]

File: Products/CMFDefault/skins/zpt_generic/main_template.pt (hunk 3 above)
Comment (rodfersou) [reply Products/CMFDefault/skins/zpt_generic/main_template.pt]: I forgot this line.. going to remove

**Comment by rodfersou** [comment]
@tseaver I'm on vacation this week.. I'll take a look at it next week if everything goes fine.

**Comment by rodfersou** [comment]
@tseaver done

**Review by tseaver — APPROVED** [review]

**Pull request merged**